apps/dashboard/src/contract-ui/tabs/listings/components/list-form.tsx

```diff
@@ -122,12 +114,15 @@ export const CreateListingsForm: React.FC<NFTMintForm> = ({
     },
   });
 
-  const { contract } = useContract(form.watch("contractAddress"));
+  const { contract: selectedContract } = useContract(
+    form.watch("contractAddress"),
+  );
   const address = useActiveAccount()?.address;
   const { data: ownedNFTs, isLoading: isOwnedNFTsLoading } = useOwnedNFTs(
```

Comment:
can't migrate this yet, right?

Reply:
correct ^ due to the difference between v5's getOwned

However I can technically write a react hook that behave similarly to v4 getownednfts if you prefer?

Reply:
no we need to use chainsaw for this